jest.config.ts

```diff
@@ -3,6 +3,7 @@ module.exports = {
   transform: {
     "^.+\\.[t|j]sx?$": "babel-jest",
   },
+  transformIgnorePatterns: ["node_modules/(?!(@runcitadel))/"],
```

Comment:
what is this doing?

Reply:
From my understanding @runcitadel is using ESM aka i.e `import` instead of i.e. `required`. Alby is also using this in its code. For Jest to handle these files Babel is being used to transpile this into something Jest can handle (same for the tsx-files).

Per default `node_modules` are not being transpiled because it can be expected that node-modules are shipping already transpiled dist code. 
[@runcitadel is using ESM](https://github.com/getAlby/lightning-browser-extension/pull/769#issuecomment-1086654095) also in the dist-files. Therefor we need to include `@runcitadel` into transpiling. 

Does this make sense? @dylancom what do you think?

Reply:
when I hear babel 🏃‍♂️😱😵‍💫 

:D